bin/build_pipeline_heka.sh

```diff
@@ -157,6 +162,20 @@ src/pkey.c src/rsa.c src/ssl.c src/th-lock.c src/util.c src/x509.c src/xattrs.c
 
 gcc -DPTHREADS -I${LUA_INCLUDE_PATH} -Ideps $SO_FLAGS $LUA_OPENSSL_SRC -lssl -lcrypto -lrt -ldl -o $HEKA_MODS/openssl.so
 
+HEKA_IO_MODS=$BASE/build/heka/build/heka/lib/luasandbox/io_modules
+mkdir -p $HEKA_IO_MODS/luasql
+echo 'Installing luasql-postgresql lib'
+cd $BASE/build
+if [ ! -d luasql-postgresql ]; then
+    git clone https://github.com/LuaDist/luasql-postgresql.git
+fi
+cd luasql-postgresql
+
+# Use a known revision (current "master" 2013-02-18)
+git checkout 29a3aa1964aeac93323ec5d1446ac7d32ec700df
+
+gcc -I/usr/include/postgresql -I${LUA_INCLUDE_PATH} $SO_FLAGS src/ls_postgres.c src/luasql.c -lpq -o $HEKA_IO_MODS/luasql/postgres.so
```

Comment:
Can we set the postgres include path to something like `$(pg_config --includedir)` ?  Do we have the `pg_config` command available when building?